packages/suite/src/components/suite/modals/confirm/Xpub/index.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { createRef } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import styled from 'styled-components';
+import * as notificationActions from '@suite-actions/notificationActions';
+import { Button, H2, colors } from '@trezor/components-v2';
+import { copyToClipboard } from '@suite-utils/dom';
+import { Dispatch } from '@suite-types';
+import { Account } from '@wallet-types';
+import { Translation, QrCode } from '@suite-components';
+import messages from '@suite/support/messages';
+
+const StyledWrapper = styled.div`
+    max-width: 600px;
+    padding: 40px;
+`;
+
+const Address = styled.div`
+    width: 100%;
+    background: ${colors.BLACK96};
+    border: 1px solid ${colors.BLACK80};
+    border-radius: 6px;
+    word-break: break-all;
+    font-size: 20px;
+    padding: 20px;
+    margin-bottom: 40px;
+`;
+
+const Row = styled.div`
+    display: flex;
+    justify-content: center;
+
+    button + button {
+        margin-top: 10px;
+    }
+`;
+
+const mapDispatchToProps = (dispatch: Dispatch) => ({
```

Comment:
Please create the container.

Reply:
i dont see a point to create Container for one-liner